var/spack/repos/builtin/packages/rocalution/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Rocalution(CMakePackage):
+    """rocALUTION is a sparse linear algebra library with focus on
+       exploring fine-grained parallelism on top of AMD's Radeon Open
+       eCosystem Platform ROCm runtime and toolchains, targeting modern
+       CPU and GPU platforms. Based on C++ and HIP, it provides a portable,
+        generic and flexible design that allows seamless integration with
+       other scientific software packages."""
+
+    homepage = "https://github.com/ROCmSoftwarePlatform/rocALUTION"
+    url      = "https://github.com/ROCmSoftwarePlatform/rocALUTION/archive/rocm-3.5.0.tar.gz"
+
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
+
+    version('3.5.0', sha256='be2f78c10c100d7fd9df5dd2403a44700219c2cbabaacf2ea50a6e2241df7bfe')
+
+    depends_on('cmake@3.5.2', type='build')
```

Comment:
```suggestion
    depends_on('cmake@3.5.2', type='build')
```
```suggestion
    depends_on('cmake@3.5:', type='build')
```
Any version newer than 3.5 should work.